### ISSUE TITLE
eslint 2.0 newline-after-var bug with brackets

### ISSUE THREAD

**Issue by evilebottnawi** [eslint/eslint#5246]
```javascript
var cond = true;

if (cond) {
    var foo = 1;
}
```
Output:
```
4:5  error  Expected blank line after variable declarations  newline-after-var
```

Rule `newline-after-var` must be ignored brackets. 
```javascript
var cond = true;

if (cond) {
    var foo = 1;
    var bar = 1;
}
```
Also with bugs.

**Comment by eslintbot** [comment]
Hi @evilebottnawi, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by ilyavolodin** [comment]
This rule enforces new line after variable declaration. As far as I can tell, it's doing the right thing here. Are you proposing an enhancement to the existing rule? Could you also answer bot's questions please?

**Comment by evilebottnawi** [comment]
@ilyavolodin 
1. 2.0.0
2. 
```javascript
var cond = true;

if (cond) {
    var foo = 1;
}
```
```json
{
    "parserOptions": {
        "ecmaVersion": 6
    },
    rules: {
        "newline-after-var": [
                2,
                "always"
            ]
    }
}
```
3. 
```
4:5  error  Expected blank line after variable declarations  newline-after-var
```
4. Ignore brackets.

**Comment by btmills** [comment]
I'm in favor of add an exception to `newline-after-var` to ignore brackets. Without the exception, we have a conflict with `padded-blocks`:

```js
/* eslint newline-after-var: 2 */
/* eslint padded-blocks: [2, "never"] */

{
    var foo;
}

{
    var bar;

}
```

```
/Users/brandon/code/eslint/eslint/test.js
   5:5  error  Expected blank line after variable declarations  newline-after-var
  11:1  error  Block must not be padded by blank lines          padded-blocks

✖ 2 problems (2 errors, 0 warnings)
```

**Comment by alberto** [comment]
Agreed.
